[2024-01-18T22:01:19.744+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_naive scheduled__2024-01-01T01:30:00+00:00 [queued]>
[2024-01-18T22:01:19.758+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_naive scheduled__2024-01-01T01:30:00+00:00 [queued]>
[2024-01-18T22:01:19.759+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:01:19.759+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-18T22:01:19.759+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-18T22:01:19.781+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): train_model_naive> on 2024-01-01 01:30:00+00:00
[2024-01-18T22:01:19.790+0300] {standard_task_runner.py:55} INFO - Started process 5555 to run task
[2024-01-18T22:01:19.798+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'train_model_naive', 'scheduled__2024-01-01T01:30:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmp1y_k4w_1']
[2024-01-18T22:01:19.803+0300] {standard_task_runner.py:83} INFO - Job 21: Subtask train_model_naive
[2024-01-18T22:01:19.939+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.train_model_naive scheduled__2024-01-01T01:30:00+00:00 [running]> on host xflow
[2024-01-18T22:01:20.183+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=train_model_naive
AIRFLOW_CTX_EXECUTION_DATE=2024-01-01T01:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-01T01:30:00+00:00
[2024-01-18T22:01:20.189+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-18T22:01:20.191+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/angry/mlops_4/scripts/train_model.py naive']
[2024-01-18T22:01:20.269+0300] {subprocess.py:86} INFO - Output:
[2024-01-18T22:01:42.926+0300] {subprocess.py:93} INFO - Registered model 'naive_model' already exists. Creating a new version of this model...
[2024-01-18T22:01:42.988+0300] {subprocess.py:93} INFO - Created version '4' of model 'naive_model'.
[2024-01-18T22:01:43.943+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-18T22:01:43.982+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=train_model_naive, execution_date=20240101T013000, start_date=20240118T190119, end_date=20240118T190143
[2024-01-18T22:01:44.047+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-18T22:01:44.082+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
